library/std/src/sys/unix/rand.rs

```diff
@@ -41,6 +44,17 @@ mod imp {
             ) -> libc::ssize_t
         }
 
+        static GRND_INSECURE_AVAILABLE: AtomicBool = AtomicBool::new(true);
+        if GRND_INSECURE_AVAILABLE.load(Ordering::Relaxed) {
+            const GRND_INSECURE: libc::c_uint = 0x0004;
```

Comment:
It would be better to get the constant definition into the libc crate

Reply:
I did. https://github.com/rust-lang/libc/pull/2743

Eventually that'll make its way to this repo and we can use it. But for now, we define the constant there.

Reply:
The constant should be available now.